File: Python3/프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.py
def solution(n):

    # range(시작범위, n-1) 까지 반복  n은 인덱스라 생각하면됨,
    # 1부터 시작한 이유는 %를 할때 0이 있으면 오류생김
    for i in range(1,n):
        if n % i == 1:
            return i

File: Java/프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        // 나머지가 1인 가장 작은수를 찾으면 값을 전달하고 break해야함.
        for(int i = 1; i < n; i++){
            if(n % i == 1){
                answer = i;
                break;
            }
        }
        return answer;
    }
}